extensions-core/parquet-extensions/src/test/java/org/apache/druid/data/input/parquet/FlattenSpecParquetReaderTest.java

```diff
@@ -205,8 +208,11 @@ public void testFlat1FlattenSelectListItem() throws IOException
         flattenSpec
     );
     List<InputRowListPlusRawValues> sampled = sampleAllRows(reader);
+    ObjectMapper obj = new ObjectMapper();
+    JsonNode expectedJson = obj.readTree(FLAT_JSON);
 
-    Assert.assertEquals(FLAT_JSON, DEFAULT_JSON_WRITER.writeValueAsString(sampled.get(0).getRawValues()));
+    JsonNode sampledAsStringJson = obj.readTree(DEFAULT_JSON_WRITER.writeValueAsString(sampled.get(0).getRawValues()));
```

Comment:
can we minimize the code changes and variables used similar to my comment for `TimestampsParquetReaderTest.java`